rpm/temboardui.plugins.monitoring.model.orm.py.patch

```diff
@@ -1,5 +1,5 @@
---- temboard-ui-0.0.1/temboardui/plugins/supervision/model/orm.py.orig	2016-07-15 21:34:18.031001950 +0200
```

Comment:
I have no idea. See https://github.com/dalibo/temboard/blob/master/rpm/temboardui.plugins.supervision.model.orm.py.patch
Do you think that it should be left untouched?